src/js/decorators/check-if-maintenance-mode.js

```diff
@@ -0,0 +1,9 @@
+var ui = require( 'helpers/ui' );
+
+module.exports = function checkIfMaintenanceMode( banner ) {
+  var maintenance_mode = NLX.features.maintenance_mode;
+
+  if ( maintenance_mode === 'true' ) {
```

Comment:
why not `maintenance_mode === true` ? (bool vs string)

Reply:
This is in README. We do this with all booleans, it is because of the way the settings get merged into an object in our inline JavaScript. This is probably historical, because since recently we don't have inline JavaScript anymore so they should just work as booleans.

Reply:
Ok if I fix in some future release?

Reply:
yup